brainglobe_atlasapi/atlas_generation/atlas_scripts/example_mouse.py

```diff
@@ -10,58 +9,135 @@
 
 from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
 
-
-def create_atlas(working_dir, resolution):
-    # Specify information about the atlas:
-    RES_UM = resolution  # 100
-    ATLAS_NAME = "example_mouse"
-    SPECIES = "Mus musculus"
-    ATLAS_LINK = "http://www.brain-map.org"
-    CITATION = "Wang et al 2020, https://doi.org/10.1016/j.cell.2020.04.007"
-    ORIENTATION = "asr"
-
-    # Temporary folder for nrrd files download:
-    download_dir_path = working_dir / "downloading_path"
+# a working example atlas-packaging script, which makes a simplified version
+# of the Allen Mouse Brain Atlas, at 100um resolution. See `template_script.py`
+# for a starting point to package your own atlas.
+__version__ = 0  # This will make the example mouse version 1.0
+ATLAS_NAME = "example_mouse"
+CITATION = "Wang et al 2020, https://doi.org/10.1016/j.cell.2020.04.007"
+SPECIES = "Mus musculus"  # The scientific name of the species,
+ATLAS_LINK = "http://www.brain-map.org"  # The URL for the data files
+ORIENTATION = "asr"  # The orientation of the atlas
+ROOT_ID = 997  # The id of the highest level of the atlas.
+RESOLUTION = 100  # The resolution of your volume in microns.
+
+BG_ROOT_DIR = Path.home() / "brainglobe_workingdir" / ATLAS_NAME
+
+
+def download_resources():
+    """
+    Download the necessary resources for the atlas.
```

Comment:
```suggestion
    Download the necessary resources for the atlas. Here we don't, because we can out-source this to the Allen SDK in later functions.
```